lib/ts/superTokens.tsx

```diff
@@ -315,6 +316,7 @@ export default class SuperTokens {
             action: "TO_AUTH",
             showSignIn: options.show === "signin",
         });
+        redirectUrl = appendTrailingSlashToURL(redirectUrl);
```

Comment:
Instead of changing here, i would suggest that you see where in the SDK we are actually doing a redirection using history or window APIs and modify the URL there to normalise it to add a trailing slash.

This would ensure that any redirection we do (even in the future) will be consistent.

Reply:
I've just discussed this with @alisher-aituarov, and I'd prefer it if didn't normalize this if we get the redirection URL from user overrides. It might be fine for the websiteBasePath (as we know it's a basepath), but not for other links. 
E.g., I want to redirect my users to `/dashboard` after a sign-in - overriding that to redirect to `/dashboard/` is not good.

Reply:
Hmm ok. But we also need to normalise redirection to password reset page, or to email verification page. So those need to be done manually then in those locations.

Reply:
Also password reset and email verification links generated on the backend SDK will have to be normalised this way as well?